drivers/input/input_gt911.c

```diff
@@ -102,26 +104,23 @@ static int gt911_process(const struct device *dev)
 	uint16_t reg_addr;
 	uint8_t status;
 	uint8_t points;
-	struct gt911_point_reg_t pointRegs;
 	uint16_t row, col;
-	bool pressed;
+	struct gt911_point_reg_t pointRegs;
```

Comment:
hey I just noticed few styling issues with this driver:

`pointRegs` has the wrong notation style, should be `point_regs`, but also the `_t` suffix of the struct name is wrong, that's used for typedef, it should be `struct gt911_point_reg`, and the field names also have the wrong format, `lowX`, `lowSize` etc should be `low_x` low_size`...

Wouldn't bother you with this since they are not part of your change but since you went through the hassle of reformatting the file already, could you fix these as well? :-)

Reply:
Not a problem at all, I tried to make it as coherent as possible. :)